TableSchemer/AccordionScheme.swift

```diff
@@ -8,40 +8,40 @@
 
 import UIKit
 
-public class AccordionScheme<CollapsedCellType: UITableViewCell, ExpandedCellType: UITableViewCell>: BasicScheme<CollapsedCellType> {
+open class AccordionScheme<CollapsedCellType: UITableViewCell, ExpandedCellType: UITableViewCell>: BasicScheme<CollapsedCellType> {
     
-    public typealias AccordionConfigurationHandler = (cell: ExpandedCellType, index: Int) -> Void
-    public typealias AccordionSelectionHandler = (cell: ExpandedCellType, scheme: AccordionScheme, selectedIndex: Int) -> Void
+    public typealias AccordionConfigurationHandler = (_ cell: ExpandedCellType, _ index: Int) -> Void
```

Comment:
You actually get the following error if you try to put an argument label there:

`Function types cannot have argument label '[someLabel]'; use '_' insetad`